src/extra/stream/client.ts

```diff
@@ -97,4 +97,39 @@ export class StreamClient extends EventDispatcher implements Stream {
       this.streams.delete(name);
     }
   }
+
+  private getStreamTimeout = 30_000;
```

Comment:
30 seconds is completely arbitrary, but I think we should have SOME timeout to prevent this functions from hanging if a corresponding stream never arrives for some reason

Reply:
do you have a sense of how long streams usually take to return? 30s feels like a very long time if it wasn't gonna arrive anyway. (wondering if we can shorten the timeout)

Reply:
[nit] naming a const `getStreamTimeout` is a little confusing, and also should it live up with other properties?